app/Community/Actions/LoadThinActivePlayersListAction.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Community\Actions;
+
+use App\Enums\Permissions;
+use App\Models\User;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\Cache;
+
+class LoadThinActivePlayersListAction
+{
+    /**
+     * @return array<array{
+     *   user_id: int,
+     *   game_id: int,
+     *   username: string,
+     *   display_name: string,
+     *   rich_presence: string,
+     *   game_title: string,
+     * }>
+     */
+    public function execute(
+        int $lookbackMinutes = 10,
+        int $minimumPermissions = Permissions::Registered
+    ): array {
+        return Cache::flexible(
+            "all-active-players:{$lookbackMinutes}:{$minimumPermissions}",
+            [20, 60],
+            function () use ($lookbackMinutes, $minimumPermissions) {
+                $timestampCutoff = Carbon::now()->subMinutes($lookbackMinutes);
+
+                $activePlayers = User::select([
+                    'ID as user_id',
```

Comment:
I imagine this is copied from somewhere else, but if you're mapping the values already, is the alias still desirable?

Reply:
Good point - removed `as user_id` in latest.